_episodes/interviews/2023-09-11-joel-marcey-pitching-rust.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Pitching Rust to decision-makers, with Joel Marcey"
+date: 2023-09-11T13:57:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2023-09-11-joel-marcey.mp3
+duration: "41:05"
+length: "40043596"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+Marco Otto-Witte discusses how to pitch Rust to decision-makers with [Joel Marcey](https://joelmarcey.com/), the Director of Technology at the Rust Foundation.  
+
+### Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+- [@00:00] - Introduction
+    - [Video recording of the interview](https://mainmatter.com/blog/2023/08/01/pitching-rust-to-decision-makers-with-joel-marcey/)
+- [@00:33] - Start of the interview
+- [@00:50] - Joel’s role at the Rust Foundation
+- [@03:17] - The value of Rust adoption for companies
+- [@07:29] - Real-world success stories with Rust
+- [@11:34] - Benefits and potential downsides of Rust adoption
+- [@18:17]- Rust’s competitive advantage for hiring
+- [@20:18] - Rust’s benefits
```

Comment:
How's this different from "benefits of Rust adoption"?

Reply:
The two segments largely overlap in terms of content focus (why is Rust beautiful) but the former comes at it from the angle of "does it make sense for my business" while the latter is a bit more generic. Section naming is hard 😅

Reply:
In cases like that, I find that being more specific. So how about:
> The implications of adopting Rust at a company
and
> Where Rust shines
maybe?

Reply:
Tweaked!